image-service/src/server.ts

```diff
@@ -0,0 +1,45 @@
+import restify from "restify";
+import errs from "restify-errors";
+import {storage} from "./app";
+import logger from "./logger";
+
+const server = restify.createServer({ name: "image-server" });
+
+server.use(restify.plugins.bodyParser({
+    keepExtensions: true,
+    multiples: true,
+}));
+
+server.post("/images/upload", async (request, response, next) => {
+    if (!request.files) {
+        return next(new errs.InvalidArgumentError("No files in request"));
+    }
+    try {
+        for (const key in request.files) {
+            if (request.files[key] instanceof Array) {
+                // @ts-ignore
+                for (const file of request.files[key]) {
+                    const newPath = await storage.put(file);
```

Comment:
newPath is not used below, 
```
await storage.put(file);
```

Reply:
and in line 26

Reply:
@webwin0 It's for later use, when we will save image info to DB.